selfdrive/car/tests/routes.py

```diff
@@ -119,6 +119,7 @@ class CarTestRoute(NamedTuple):
   CarTestRoute("36e10531feea61a4|2022-07-25--13-37-42", HYUNDAI.TUCSON_HYBRID_4TH_GEN),
   CarTestRoute("5875672fc1d4bf57|2020-07-23--21-33-28", HYUNDAI.KIA_SORENTO),
   CarTestRoute("1d0d000db3370fd0|2023-01-04--22-28-42", HYUNDAI.KIA_SORENTO_4TH_GEN, segment=5),
+  CarTestRoute("fc19648042eb6896|2023-08-16--11-43-27", HYUNDAI.KIA_SORENTO_HEV_4TH_GEN, segment=14),
```

Comment:
what happens in segment 14?

Reply:
I figure it might be a better segment to use in test routes and doesn't fail CI, CI said segments 0-4 were missing logs (which it isn't).

Reply:
They simply weren't uploaded to the CI bucket yet